core/kazoo_apps/src/kapps_maintenance.erl

```diff
@@ -402,8 +402,8 @@ refresh_account_db(Database) ->
     AccountId = kz_util:format_account_id(Database, 'raw'),
     _ = remove_depreciated_account_views(AccountDb),
     _ = ensure_account_definition(AccountDb, AccountId),
-    Views = get_all_account_views(),
-    _ = kapps_util:update_views(AccountDb, Views, 'true'),
+    _ = kapps_util:update_views(AccountDb, get_all_account_views(), 'true'),
+    _ = knm_maintenance:update_number_services_view(AccountDb),
```

Comment:
There's [a TODO](https://github.com/2600hz/kazoo/pull/2748/files#diff-61f44c46777bb6f0a3bd051aafcff1eeR45) that asks for per-account classifiers.
If we do it this way for now, we will have to move things around again when we implement the TODO.
What do you think?